src/app.md

```diff
@@ -271,6 +272,40 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot]: #ex-file-skip-dot
+<a name="ex-file-skip-dot"></a>
+##  Traverse directories while skipping dotfiles
+
+Iterates over filesystem entries efficiently skipping hidden files and directories with help of [WalkDir::filter_entry]. 
+
+[`WalkDir::filter_entry`] descends recursively into entries passing the `is_not_hidded` predicate. [`std::iter::Iterator::filter`] would be applied to each `DirEntry` even if the parent is a hidden directory. Uses [`WalkDir::depth`] to avoid skipping root directory `"."` at level 0.
```

Comment:
[`std::iter::Iterator::filter`] is a little to long I would souggest [`Iterator::filter`].

Also I would suggest to join the two sentences with 
"into entries passing the `is_not_hidded` predicate whereas [`Iterator::filter`] would be applied"